DynmapCore/src/main/java/org/dynmap/DynmapWorld.java

```diff
@@ -154,6 +155,7 @@ private void processZoomFile(MapTypeState mts, MapStorageTile tile, boolean firs
             }
             try {
                 MapStorageTile.TileRead tr = tile1.read();
+                mostRecentTimestamp = Math.max(mostRecentTimestamp, tr.lastModified);
```

Comment:
Now getting this exception:
```
[11:05:10 ERROR]: [dynmap] Exception during render job: org.dynmap.MapManager$DoZoomOutProcessing@677cad04
[11:05:10 WARN]: java.lang.NullPointerException: Cannot read field "lastModified" because "tr" is null
[11:05:10 WARN]:        at org.dynmap.DynmapWorld.processZoomFile(DynmapWorld.java:158)
[11:05:10 WARN]:        at org.dynmap.DynmapWorld.freshenZoomOutFiles(DynmapWorld.java:114)
[11:05:10 WARN]:        at org.dynmap.MapManager$DoZoomOutProcessing.run(MapManager.java:940)
[11:05:10 WARN]:        at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:245)
[11:05:10 WARN]:        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[11:05:10 WARN]:        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[11:05:10 WARN]:        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
[11:05:10 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[11:05:10 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[11:05:10 WARN]:        at java.base/java.lang.Thread.run(Thread.java:831)
```

Reply:
Not all subtiles necessarily exist ...
You could do `Math.max(mostRecentTimestamp, tr==null?0:tr.lastModified);`

Reply:
I know how to fix it ;)  
Just wanted to note it down when I saw it, because that's all that I could do in that moment.